setup-machine-arktos.sh

```diff
@@ -5,7 +5,19 @@
 echo Setup: Install go \(currently limited to version 1.13.9\)
 
 sudo apt-get update -y -q
-sudo apt-get install libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev -y -q
+sudo apt-get install build-essential -y -q
+sudo apt-get install zlib1g-dev -y -q
```

Comment:
Those packages were added because compiling python from source code requires additional packages i.e build-essential, zlib1g-dev and others. I did not remove build-essential from later because after updating python version from 3.6 to 3.8, there might be some package which might get updated according to python version.